doc/dynamo0p3.rst

```diff
@@ -1246,35 +1246,35 @@ where:
 Boundary Conditions
 -------------------
 
-In the dynamo0.3 API, boundary conditions for a field can be enforced
-by the algorithm developer by calling a particular Kernel called
-``enforce_bc_type``. This kernel takes a field as input and applies
+In the dynamo0.3 API, boundary conditions for a field or LMA operator can
```

Comment:
At some point we should decide that this is no longer the 0.3 version of the API...maybe 1.0?